.travis.yml

```diff
@@ -0,0 +1,33 @@
+language: c
```

Comment:
Maybe cpp? It's actually both, but I don't know if it matters :)

Reply:
I think that's used to pick the tools installed by default. It's a single value, but `cpp` probably includes all the C tools too so I'll set it to that :+1: